File: level1/Python/[1차] 비밀지도.py
def solution(n, arr1, arr2):
    answer = []

    for i in range(n):
        ls=""
        a=format(arr1[i],'b').zfill(n)
        b=format(arr2[i],'b').zfill(n)
        
        for j in range(len(a)):
            if a[j]=="1" or b[j]=="1":
                ls+="#"
            else:
                ls+=" "
        answer.append(ls)
        
    return answer

File: level1/C++/[1차] 비밀지도.cpp
def solution(n, arr1, arr2):
    answer = []
    
    for i in range(n):
        line1=format(arr1[i],"b").zfill(n)
        line2=format(arr2[i],"b").zfill(n)
        ss=""
        for j in range(n):
            if line1[j] == "1" or line2[j] =="1":
                ss+="#"
            else:
                ss+=" "
        answer.append(ss)
    
    
    
    return answer
